docs/site/src/app/pages/enterprise/enterprise.component.scss

```diff
@@ -3,17 +3,137 @@
     min-height: 200px;
   }
 
+  .info {
+    font-weight: unset;
+    text-align: center;
+    line-height: 30px;
+    max-width: 800px;
+    margin: 20px auto;
+  }
+
+  p {
+    color: #1c1c50;
+    font-size: 1rem;
+    font-weight: 100;
+    line-height: 1.7;
+    text-align: justfiy;
+    padding-top: 12px;
+  }
+
+  a {
+    display: unset !important;
+  }
+
+  // Text Properties
+  .text-black {
+    color: black;
+  }
+
+  .text-gray {
+    color: #263238;
+  }
+
+  .text-white {
+    color: white;
+  }
+
+  .text-default {
+    color: #78909c;
+  }
+
+  .plugin {
+    border-radius: 4px !important;
+    -webkit-box-shadow: 0 6px 12px -2px rgba(50, 50, 93, 0.25), 0 3px 7px -3px rgba(0, 0, 0, 0.3);
+    box-shadow: 0 6px 12px -2px rgba(50, 50, 93, 0.25), 0 3px 7px -3px rgba(0, 0, 0, 0.3);
+    margin: 10px !important;
+    padding: 40px !important;
+    transition: all 1s;
+    min-height: 150px;
+    background: #fff !important;
+
+    h3 {
+      font-size: 150%;
+    }
+
+    p {
+      padding-left: 30px;
+      padding-right: 30px;
+    }
+
+    .price {
+      border-left: 1px solid #eee;
+    }
+
+    h3 {
+      vertical-align: middle;
+      margin: 0;
+      padding-bottom: 12px;
+      font-weight: normal;
+
+      mat-icon {
+        vertical-align: bottom;
+      }
+    }
+  }
+
+  // Margin && Padding
+
+  .unset-margin {
+    margin: unset !important;
+  }
+
+  .pt-0 {
+    padding-top: 0 !important;
+  }
+
+  .unset-padding {
+    padding: unset !important;
+  }
+
+  .blue {
+    background-color: #2196f3;
+  }
+
+  .gray {
+    background-color: #263238;
+  }
+
+  .bg-gray {
+    background: #eee;
+  }
+
+  .bg-black {
+    background: #263238;
+  }
+
+  .contact-button {
+    display: block;
+    margin: 0 auto;
+  }
```

Comment:
Avoid styling components via classes instead you should tightly couple them to their styles by specifying more strict selectors.